discord/webhook/async_.py

```diff
@@ -1204,8 +1204,6 @@ async def edit(
             Whether to use the bot token over the webhook token
             if available. Defaults to ``True``.
 
-            .. versionadded:: 2.0
-
```

Comment:
Was this not added in 2.0?

Reply:
It also has added in 1.6 above and there wasn’t any change notes here so I removed it